content/1_General/Debugging_General.mdx

```diff
@@ -6,24 +6,29 @@ description: 'General tips for identifying errors within your program.'
 ---
 
 <Resources>
-	<Resource source="KACTL" title="Troubleshooting" url="https://github.com/kth-competitive-programming/kactl/blob/master/content/contest/troubleshoot.txt" starred>
+	<Resource
+		source="KACTL"
+		title="Troubleshooting"
+		url="https://github.com/kth-competitive-programming/kactl/blob/master/content/contest/troubleshoot.txt"
+		starred
+	>
 		things to try in an ICPC contest
 	</Resource>
 	<Resource
 		source="Errichto"
 		title="Asking for help FAQ"
 		url="https://codeforces.com/blog/entry/64993"
 		starred
-	>
-	</Resource>
+	/>
 </Resources>
 
 This module is based on both of these resources. I've included the content that is most relevant to USACO.
 
 ## Pre-Submit
 
- - Don't repeat yourself!
+- Don't repeat yourself!
 
+<!--
 <LanguageSection>
 
 <JavaSection>
```

Comment:
might as well just delete the entire thing instead of commenting it out -- it can be found via the PR (https://github.com/cpinitiative/usaco-guide/pull/823/files) anyway